add-steps/linkerd.yml

```diff
@@ -0,0 +1,37 @@
+# use the filesystem namer, read from the local /disco directory
+namers:
+- kind: io.l5d.fs
+  rootDir: /disco
+
+routers:
+- protocol: http
+  servers:
+  - port: 4140
+    ip: 0.0.0.0
+  # route all traffic to service identified in service discovery as "app"
+  baseDtab: >-
+    /http/* => /#/io.l5d.fs/app;
+  client:
+    loadBalancer:
+      kind: ewma
+    failureAccrual:
+      kind: io.l5d.successRate
+      successRate: 0.99
+      requests: 20
+      backoff:
+        kind: jittered
+        minMs: 10000
+        maxMs: 20000
+  #   # This section left here for demonstration.
+  #   # Uncommenting this will result in 100% success rate at the expense of latency.
+  #   retries:
+  #     budget:
+  #       minRetriesPerSec: 5
+  #       percentCanRetry: 0.5
+  #       ttlSecs: 15
+  #     backoff:
+  #       kind: jittered
+  #       minMs: 10000
+  #       maxMs: 20000
+  # responseClassifier:
+  #   kind: io.l5d.retryableRead5XX
```

Comment:
TIOLI I tried re-enabling the retryableRead5XX responseClassifier locally, and it totally works. Latency is still lower, and success rate is 100%. I think it might be worth adding this back into the config, to show a comprehensive tuning example that includes retries.

![screen shot 2017-01-27 at 4 57 06 pm](https://cloud.githubusercontent.com/assets/9226/22392758/b5218a16-e4b1-11e6-9356-73b01787fce4.png)